kubernetes/DEMO.md

```diff
@@ -137,13 +138,13 @@ logout
 
 Create network policy object to allow access from nginx-frontend to nginx-backend
 ```
-root@ip-192-168-0-10:/home/ubuntu# curl -X POST -d @romana/kubernetes/romana-network-policy-request.json http://192.168.0.10:8080/apis/romana.io/demo/v1/namespaces/default/networkpolicys/
+root@ip-192-168-0-10:/home/ubuntu# curl -X POST -d @romana/kubernetes/romana-network-policy-request.json http://192.168.0.10:8080/apis/romana.io/demo/v1/namespaces/tenant-a/networkpolicys/
 {
   "apiVersion": "romana.io/demo/v1",
   "kind": "NetworkPolicy",
   "metadata": {
     "name": "pol1",
-    "namespace": "default",
+    "namespace": "tenantA",
```

Comment:
"tenant-a" or "tenantA" is both fine, thank you for changing this.

But just out of curiosity, how come we use "tenant-a" in one place and "tenantA" in the other? I understand that they do a funny camel-case conversion driven by dashes in the string. But, for example, if the namespace name in 'isolated-namespace.yml' would have been specified as 'tenantA' instead of 'tenant-a', would the URL have been '.../tenantA/...' ?

It bugs me a little that during the demo we might have to point this out, or have some people confused by that.